internal/db/pgsql/orgs.go

```diff
@@ -91,15 +91,15 @@ func (p *Provider) GetOrg(ctx context.Context, id uint64) (*model.Organization,
 func (p *Provider) RemoveOrg(ctx context.Context, id uint64) error {
 	_, err := p.db.ExecContext(ctx, `DELETE FROM orgmembers WHERE org_id=$1;`, id)
 	if err != nil {
-		logger.Errorf("api=RemoveOrg, err=[%s]", errors.Details(err))
```

Comment:
just curious, any particular reason "api=" is removed? Seems useful to have it. Or are we saying it will anyways be in the stack trace of the details?

Reply:
Now logger package add `src=FuncName` automatically, so we don't need to add it in every log